src/kuzya.cpp

```diff
@@ -659,6 +659,11 @@ void Kuzya::slotSetFileSuffix(QStringList filter)
     }
 }
 
+void Kuzya::slotSetFileSuffix(QString suffix)
+{   //this methods explicitly converts QString to QstrinList. Using by singals/slot connection
+    slotSetFileSuffix(static_cast<QStringList>(suffix));
```

Comment:
It should work. I tested it. QStringList has constructor with QString parametr. static_cast uses this, i think.